versions/a-/am-midi2.json

```
@@ -0,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "399feb8cb4ed84cc506710d6ca63e51670528679",
+      "version": "0.6",
```

```suggestion
      "version": "0.7",
```
Please fix this and re-run `./vcpkg x-add-version --all` to verify.

---

@WangWeiLin-MV sorry I missed this - I thought I had it.
I deleted versions/a-/am-midi2.json and it now created versions/l-/libmidi2.json as renamed the port (as per suggestions)